charts/gardener/operator/templates/clusterrole.yaml

```diff
@@ -24,26 +24,21 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - events
+  - namespaces
   verbs:
   - create
+  - get
+  - list
+  - watch
```

Comment:
We could restrict all verbs on `namespaces` to the release namespace, right?

Reply:
Without this, the operator will error with

```
2022-11-21T08:08:07.663+0100	ERROR	Reconciler error	{"controller": "garden", "controllerGroup": "operator.gardener.cloud", "controllerKind": "Garden", "Garden": {"name":"garden"}, "namespace": "", "name": "garden", "reconcileID": "b3739976-b0f1-4750-94c2-2cc2ba11adb7", "error": "namespaces is forbidden: User \"system:serviceaccount:garden:gardener-operator\" cannot create resource \"namespaces\" in API group \"\" at the cluster scope"}
...
2022-11-21T08:08:40.297+0100	ERROR	sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:262: Failed to watch *v1.Namespace: failed to list *v1.Namespace: namespaces is forbidden: User "system:serviceaccount:garden:gardener-operator" cannot list resource "namespaces" in API group "" at the cluster scope
```

Reply:
Right, we could run the manager against the `garden` namespace only.
But let's keep it, for now – it doesn't really matter as long as we have `escalate` privileges.

Reply:
> Right, we could run the manager against the garden namespace only.

This should be already done: https://github.com/rfranzke/gardener/blob/4822b5311b480570008df0523b4cc084931882ef/cmd/gardener-operator/app/app.go#L117 👀

Reply:
Ah, right 👀 
`manager.Options.Namespace` only restricts the cache of namespace-scoped resources, but not those of cluster-scoped resources.
If you wanted to watch only the garden namespace, you would need to add a dedicated selector to the namespace cache.